### DISCUSSION TITLE
If PODs text field has a shortcode, then it won't work with the magic tag output filters

### DISCUSSION THREAD

**Discussion by wisamx** [pods-framework/pods#6653] [Q&A]
**Describe the bug**
I have the text field 'credits' which has a pods shortcode (to show a field value from another pods entity field).
'credits' field value:

`[pods   name='contentblock' slug='credits-bba' field='value']`

in the pods template I used a magic tag with a filter function, to work on the value of the field (generated through shortcode):

```
                                        [if credits]
                                        <li class="credits-feature">
                                            <span class="value">{@credits, pods_style_the_credits}</span>
                                        </li>
                                        [else][/if]
```

**Expected behavior**
the shortcode should be compiled returning the data, so the output filter process this data.

**what is going**
seems like the output filter work before the shortcode being compiled.

## Pods Version
latest

**Comment by wisamx** [comment]
I tried this filter function:

```
function pods_style_the_credits($value) {
	echo $value;
	die();
}
```

I get the shortcode printed without being compiled.
I want the shortcode to be compiled first, then apply the output function!

Thanks!

**Comment by JoryHogeveen** [comment]
Hello @wisamx 

Please file a complete bug report, not just the description.
I'd like to have a bit more information on where you exactly use what code and in what order.

Am I correct in that `{@credits}` would be a shortcode itself (the value of this field). If so, that cannot be done. You'll have to wrap it in `apply_shortcodes()` when parsing it through `pods_style_the_credits()`.
https://developer.wordpress.org/reference/functions/apply_shortcodes/

Cheers, Jory
